app/views/devise/sessions/new.html.erb

```diff
@@ -1,28 +1,23 @@
 <div class='login-section'>
     <h2>Log in</h2>
```

Comment:
- [ ] Your form on the sign up page is aligned to the left. Kindly position the form on the sign up page to be centered like the login and the wireframe layout.

  ![image](https://github.com/maths29/blog_app/assets/36542986/9c3e28b0-0577-4854-9507-d7cab083fdf7)